### PR TITLE
feat(tree-sitter): add sql language

### PR DESCRIPTION
**Pull request by oknozor**
- [x] Added an entry to `CHANGELOG.md` if this change could be valuable to users
- Partially address #272

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,14 +2,10 @@
 
 ## Unreleased
 
-### Features/Changes
```
Comment (dzhou121) [inline CHANGELOG.md:5]: why did you delete these titles

Comment (oknozor) [reply CHANGELOG.md:5]: I messed up rebasing, fixed now, sorry

**Comment by dzhou121** [comment]
Can you please fix changelog again :)
